src/components/socials.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from "styled-components"
+
+import Link from "./link"
+
+// Import social logos
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faFacebookSquare } from '@fortawesome/free-brands-svg-icons'
+import { faInstagramSquare } from '@fortawesome/free-brands-svg-icons'
+import { faLinkedin } from '@fortawesome/free-brands-svg-icons'
+
+
+const MhpSocialContainer = styled.div`
+    display: block;
+    text-align: center;
+`
+
+const SocialLink = styled(Link)`
+    padding:  0px 10px 0px 10px;
+    font-size: 28px;
+    color: var(--MHP-white);
+
+    &:hover {
+        color: var(--MHP-green);
+        transition: 0.3s;
+    }
+`
+
+const Socials = () => {
+    return (
+        <MhpSocialContainer>
+            <SocialLink to='https://www.facebook.com/MonashHPT/'>
```

Comment:
Might want to do that refactor here as well. `const socialLink = [...]` etc. etc.

This set's us up later so that corporate is able to edit this using a CMS.

Reply:
Because the `FontAwesomeIcon` is needed to be imported I think I will leave it for another PR as it will be more intuitive to integrate it together once the CMS is running.

Reply:
Sounds good to me!